handlers/workspaces.go

```diff
@@ -512,16 +512,16 @@ func (oh *workspaceHandler) GetUserDropdownWorkspaces(w http.ResponseWriter, r *
 	json.NewEncoder(w).Encode(workspaces)
 }
 
-func GetCreatedWorkspaces(pubkey string) []db.Workspace {
-	workspaces := db.DB.GetUserCreatedWorkspaces(pubkey)
+func (oh *workspaceHandler) GetCreatedWorkspaces(pubkey string) []db.Workspace {
+	workspaces := oh.db.GetUserCreatedWorkspaces(pubkey)
 	// add bounty count to the workspace
 	for index, value := range workspaces {
 		uuid := value.Uuid
-		bountyCount := db.DB.GetWorkspaceBountyCount(uuid)
-		hasRole := db.UserHasAccess(pubkey, uuid, db.ViewReport)
+		bountyCount := oh.db.GetWorkspaceBountyCount(uuid)
+		hasRole := oh.userHasAccess(pubkey, uuid, db.ViewReport)
```

Comment:
UserHasAccess is in the Database interface so you can change it to `oh.db.UserHasAccess`, let me know if this fixes it.

Reply:
@elraphty The same error still persists. I tried everything, but the UserHasAccess error is not fixed.

Reply:
Push your latest changes, let me give it a look